spec/observables/empty-spec.ts

```diff
@@ -1,16 +1,42 @@
-import * as Rx from '../../src/Rx';
 import marbleTestingSignature = require('../helpers/marble-testing'); // tslint:disable-line:no-require-imports
+import { empty } from '../../src/create';
+import { expect } from 'chai';
 
-declare const { asDiagram };
+declare const asDiagram: any;
 declare const expectObservable: typeof marbleTestingSignature.expectObservable;
-
-const Observable = Rx.Observable;
+declare const rxTestScheduler: any;
 
 /** @test {empty} */
-describe('Observable.empty', () => {
+describe('empty', () => {
   asDiagram('empty')('should create a cold observable with only complete', () => {
     const expected = '|';
-    const e1 = Observable.empty();
+    const e1 = empty();
     expectObservable(e1).toBe(expected);
   });
+
+  it('should return the same instance EMPTY', () => {
+    const s1 = empty();
+    const s2 = empty();
+    expect(s1).to.equal(s2);
```

Comment:
since we export `EMPTY` constant, maybe `expect(s1).to.equal(s2).to.equal(EMPTY)`?

Reply:
This test, combined with the test I just added (`empty()` equals `EMPTY`) should do the trick.